e2e-tests/cypress/integration/appellant-submission-decision-letter.feature

```diff
@@ -0,0 +1,39 @@
+@wip
+Feature: Appellant submission - decision letter
+
+  An appeal must include an uploaded decision letter file.
+  Valid decision letter files must:
+  * be one of the white-listed types (doc, docx, pdf, tif, tiff, jpg, jpeg and png) and
+  * not exceed a size limit.
+  The latest successfully uploaded decision letter file replaces any previously uploaded file.
+
+  Scenario Outline: Prospective appellant submits valid decision letter file
+    When user submits a decision letter file <filename>
```

Comment:
minor point; are we happy with the economy of losing 'the' in these? i sort of don't mind and sort of do?

Reply:
Thanks. I think we can work together to get some guidance out so that we phrase our features files more consistently.